build.gradle

```diff
@@ -63,6 +63,62 @@ test {
     }
 }
 
+def batchProcessorIntegrationTestTask = tasks.register('batchProcessorIntegrationTest', JavaExec) {
+    group 'Verification'
+    classpath = sourceSets.test.runtimeClasspath
+    mainClass.set 'marytts.BatchProcessor'
+    def batchFile = file("$temporaryDir/batch.json")
+    args batchFile
+    systemProperty 'mary.base', temporaryDir
+    if (JavaVersion.current().java9Compatible) {
+        jvmArgs = [
+                '--add-opens', 'java.base/java.io=ALL-UNNAMED',
+                '--add-opens', 'java.base/java.lang=ALL-UNNAMED',
```

Comment:
At first, it seems we also need to add
```java
                '--add-opens', 'java.base/java.lang.reflect=ALL-UNNAMED',
```
but the problem is more subtle, since this causes a processing failure within MaryTTS... From the relevant `server.log`:
```
2023-01-11 04:58:15,004 [main] ERROR marytts.BatchProcessor Unable to make public com.sun.org.apache.xerces.internal.dom.DocumentImpl(boolean) accessible: module java.xml does not "exports com.sun.org.apache.xerces.internal.dom" to unnamed module @7a92922
```

Reply:
The solution here is to ensure that a proper XML parser is on the classpath.

Reply:
Resolved via 445b4442a312eaa137fcc18fb9e25e3d59ef7abf